Shared/Extensions/JellyfinAPIExtensions/MediaStreamExtension.swift

```diff
@@ -10,12 +10,8 @@ import Foundation
 import JellyfinAPI
 
 extension MediaStream {
-
 	func externalURL(base: String) -> URL? {
 		guard let deliveryURL = deliveryUrl else { return nil }
-		var baseComponents = URLComponents(string: base)
-		baseComponents?.path += deliveryURL
-
-		return baseComponents?.url
+		return URL(string: base + deliveryURL)
```

Comment:
Great find! However, we are not guaranteed that the base url does not end in a trailing `/`, and just simply appending it would break it. A check should be made first whether `base` ends in a trailing slash, removing it from the end of`base` or the first slash from `deliveryURL`, and then creating the URL.